sql/5_0_2-to-5_0_3_upgrade.sql

```diff
@@ -439,3 +439,7 @@ ALTER TABLE `form_eye_mag_prefs` MODIFY `ordering` smallint(6) DEFAULT NULL;
 #IfNotColumnType codes code_text_short varchar(255)
 ALTER TABLE `codes` MODIFY `code_text_short` varchar(255) NOT NULL default '';
 #EndIf
+
+#IfNotColumnType amendments created_time timestamp
```

Comment:
I am willing to bet this won't work, since it is timestamp already. What if you try:
```
#IfNotColumnType amendments created_time timestamp NULL
```
(we will need to test this)

Reply:
:game_die:

Reply:
Updated!!